src/main/java/Task.java

```diff
@@ -0,0 +1,38 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
```

Comment:
The spacing between Task and '{' should be consistent with other classes as well as the line break before your 2nd line of code